Testreglar/214/Nett/nett-2.1.4a.json

```diff
@@ -0,0 +1,195 @@
+{
+	"id": "nett-2.1.4a",
+	"type": "web",
+	"kravTilSamsvar": "<p>Krav til samsvar: </p>\r\n<ul>\r\n<li>Det er en mekanisme for å slå av hurtigtasten\r\n<ul>\r\n<li>Mekanismen ligger på samme nettside som innholdet som benytter hurtigtaster eller</li>\r\n</ul>\r\n</li>\r\n<li>Det er en mekanisme for å rekonfigurere (omadressere) hurtigtasten ved hjelp av en eller flere modifiseringstaster\r\n<ul>\r\n<li>Mekanismen ligger på samme nettside som innholdet som benytter hurtigtaster eller</li>\r\n</ul>\r\n</li>\r\n<li>Hurtigtasten for en brukergrensesnittkomponent er kun aktiv når komponenten kommer i fokus</li>\r\n</ul>",
```

Comment:
Krav til samsvar må eksplisitt opplyse om disse alle punktene eller et av de må være oppfylte for å få samsvar, sL If a **keyboard shortcut** is implemented in content using only letter (including upper- and lower-case letters), punctuation, number, or symbol characters, then **at least one of the following is true**:

Reply:
Det står "eller" på slutten av hvert punkt. Hva mener du @Sivzilla82?

Reply:
@adilsofficial og @geirsf De svarte hovedpunktene er alternativer (Jeg har lagt til **og** for å knytte underpunktet sammen med det svarte hovedpunktet (delkrav som må oppfylles) Det manglet tidligere **og** mellom hovedkravet og delkravet. Tolkningen er oppdatert nå. 

•	Det er en mekanisme for å slå av hurtigtasten **og**
o	Mekanismen ligger på samme nettside som innholdet som benytter hurtigtaster eller
•	Det er en mekanisme for å rekonfigurere (omadressere) hurtigtasten ved hjelp av en eller flere modifiseringstaster **og**
o	Mekanismen ligger på samme nettside som innholdet som benytter hurtigtaster eller
•	Hurtigtasten for en brukergrensesnittkomponent er kun aktiv når komponenten kommer i fokus

Reply:
Oppdatere krav til samsvar med overnevnte setning.